app/models/ems_folder.rb

```diff
@@ -195,4 +209,11 @@ def self.child_folder_paths_recursive(subtree, options = {})
     end
   end
   private_class_method :child_folder_paths_recursive
+
+  def self.folder_options(folders, options)
```

Comment:
don't think it is modifying the `options` or `folders`. like calling it `apply_folder_options`